lib/AppInfo/Application.php

```diff
@@ -122,6 +122,7 @@ private function addAltLogins($providersType)
                 \OC_App::registerLogIn([
                     'name' => $provider['title'],
                     'href' => $this->providerUrl,
+                    'style' => $provider['style'],
```

Comment:
Should be `'style' => isset($provider['style']) ? $provider['style'] : '',` to prevent "undefined index 'style'" warnings after update

Reply:
Changed as suggested :-)